.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
-*                 @Datadog/rum-browser @Datadog/rum-mobile @Datadog/rum-back
+*                 @Datadog/rum-browser @Datadog/rum-mobile @Datadog/rum-backend
```

Comment:
The team in Github is still named `rum-back`, is this supposed to change at some point?